gtsam/navigation/CombinedImuFactor.h

```diff
@@ -61,10 +63,19 @@ struct GTSAM_EXPORT PreintegrationCombinedParams : PreintegrationParams {
   Matrix3 biasOmegaCovariance;  ///< continuous-time "Covariance" describing gyroscope bias random walk
   Matrix6 biasAccOmegaInt;     ///< covariance of bias used for pre-integration
 
+  /// Default constructor makes unitialized params struct.
+  /// Used for serialization.
+  PreintegrationCombinedParams()
+      : PreintegrationParams(),
```

Comment:
Not needed with empty args

Reply:
Gotcha. Just felt better to be consistent in our coding standards.

Reply:
Do fix